src/name/name.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright 2015-2020 Brian Smith.
```

Comment:
Since I don't think this module was authored by Brian, it probably shouldn't carry his copyright?

Reply:
I kept the same header to be consistent with the rest of the project (along the lines of "Brian Smith et al"). I'm updating the PR though.

Reply:
Addressed in https://github.com/rustls/webpki/pull/21.